ignite/metrics/negative_log_likelihood.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import division
+
+from torch.autograd import Variable
+from torch.nn.functional import nll_loss
+
+from .metric import Metric
+from ignite.exceptions import NotComputableError
+
+
+class NegativeLogLikelihood(Metric):
+    def __init__(self):
+        self._sum = 0
+        self._num_examples = 0
+
+    def reset(self):
+        self._sum = 0
+        self._num_examples = 0
+
+    def update(self, output):
+        y_pred, y = output
+        self._sum += nll_loss(Variable(y_pred), Variable(y)).data[0]
```

Comment:
It might actually be better to not average in `nll_loss`, but instead pass `size_average=False`, so that the division can be done in `compute`.